component_down.py

```diff
@@ -0,0 +1,33 @@
+import os
+import time
+import json
+from collections import defaultdict
+
+agent= os.getenv('HOSTNAME')
+now = time.mktime(time.gmtime())
+since = 7 ## in days
+
+base_dir = '/data/srv/wmagent/current/install/wmagent/'
+restarts = { 
+    "timestamp" : now,
+    "since" : since,
+    "agent" : os.getenv('HOSTNAME'),
+    "data" : defaultdict(list)
+    }
+   
+for component in os.listdir(base_dir):
+    if os.path.isfile(component):continue
```

Comment:
How about replacing L19 by:
```
if not os.path.exists(os.path.join(base_dir, component, 'ComponentLog'):
    continue
```

Reply:
I do not see an issue with this will make the change run a test and re push.